functions/exercise21/scriptSpec.js

```diff
@@ -0,0 +1,15 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals getPossibleSubset*/
+
+describe("Getting all possible subset with a fixed length combinations in an array", function () {
+    it("If the array [1, 2, 3] we will get all subsets of length 2, we get subsets [2, 1], [3, 1], [3, 2], [3, 2, 1]", function () {
+        var subsets = getPossibleSubset([1, 2, 3], 2);
+        expect(subsets).toEqual(
```

Comment:
What is the type of the return value you expect?
It is incorrectly declared in your case as you express it as multiple arguments, but toEqual can contain only one object and you specified 4 of them.